Makefile

```diff
@@ -51,6 +47,14 @@ ifeq ($(LEDGER_ENABLED),true)
   endif
 endif
 
+### Update Build flags
+BUILD_FLAGS = -tags "${BUILD_TAGS}" -ldflags \
+    '-X github.com/cosmos/cosmos-sdk/version.Version=${VERSION} \
```

Comment:
Will this fail? I _think_ variable expansion in BASH needs `""`

Reply:
No, it doesn't. It's common to do it to ensure the shell really expands vars within quotes, and also it comes handy when you want to ensure that bash treats the contents of a variable as one string only, e.g. there are differences between the following two:

```bash
list_of_things='abc def ghi'

# expands $list_of_things into a list of words
# prints:
#  abc
#  def
#  ghi
for w in $list_of_things; do echo $w ; done

# $list_of_things one-word variable
# prints:
#  abc def ghi
for w in "$list_of_things"; do echo $w ; done
```

Brackets are useful when you don't want to use double quotes but at the same time you need to avoid similar things:

```bash
echo $list_of_thingsjkl # prints empty string

echo ${list_of_things}jkl # prints abc def ghijkl
```

Reply:
BTW, Make expands variables differently than Bash. We should in fact use the "$(VARIABLE)" form instead of "${VARIABLE}" - though in this case both work.

Reply:
```$ for w in "$list_of_things"; do echo $w ; done
abc def ghi
$ for w in '$list_of_things'; do echo $w ; done
$list_of_things
```

Reply:
> I'm talking about `'` vs `"` in BASH, but it might be diff in `Makefile`s

The reason I have both ' and " is because `make` was causing trouble when nesting " and this approach was cleaner than escaping, but I can go back to "\"\"" is necessary

Reply:
It's ok, it works